wandb/sdk/wandb_run.py

```diff
@@ -336,15 +336,19 @@ def __setstate__(self, state):
 
     @property
     def dir(self):
-        """str: The directory where all of the files associated with the run are
-        placed.
+        """
+        Returns:
+            (str): The directory where all of the files associated with the run are
+                placed.
         """
         return self._settings.files_dir
 
     @property
     def config(self):
-        """(`Config`): A config object (similar to a nested dict) of key
-            value pairs associated with the hyperparameters of the run.
+        """
+        Returns:
+            (Config): A config object (similar to a nested dict) of key
```

Comment:
Will the google doc logic eventually be smart enough to know Config is a documented object?

Reply:
Cross referencing has been on my bucket list for some time now. For the time being Config is just some markdown ext for the renderer. I am looking into the doc system of tensorflow to figure out how they do it. With the present PRs, it is an effort to stabilize the doc string template. Later, working on a consistent system of docstring would be easier.